turbinia/config/recipes/triage-linux.yaml

```diff
@@ -0,0 +1,23 @@
+# Triage recipe for quick initial response (specific to Linux).
+
+globals:
+  jobs_allowlist:
+    - CronExtractionJob
+    - CronAnalysisJob
+    - PlasoJob
+    - PsortJob
+    - FileSystemTimelineJob
+
+plaso_base:
+  task: 'PlasoTask'
+
+  file_filter: [
+      '/var/log/\w+[.]log([.]\d.gz|[.]\d|$)',
```

Comment:
Should we just add all of `/var/log` here?  As is I think this will miss things like `/var/log/wtmp`, which we probably want to include.

Reply:
Agreed, i've modified the file to include all of /var/log